tools/sigh.ts

```diff
@@ -537,6 +537,21 @@ function lint(args: string[]): boolean {
   return report.errorCount === 0;
 }
 
+/** Runs buildifier on all BUILD files. */
+function buildifier(args: string[]): boolean {
+  const options = minimist(args, {
+    boolean: ['fix'],
+  });
+
+  const modeOpts = options.fix ? ['--lint=fix', '--mode=fix'] : ['--lint=warn', '--mode=check'];
+
+  return saneSpawnSync('find', [
+    '.',
+    '-name', 'BUILD', '-o', '-name', 'BUILD.bazel', '-o', '-name', 'WORKSPACE', '-o', '-name', '*.bzl',
+    '-exec', 'node_modules/@bazel/buildifier/buildifier.js', ...modeOpts, '--warnings=-module-docstring,-bzl-visibility', '{}', '\\;'
```

Comment:
maybe `npx buildifier` would be sufficient?

Reply:
Thanks, that works!